packages/sign_in_with_apple/sign_in_with_apple_platform_interface/lib/exceptions.dart

```diff
@@ -114,6 +129,15 @@ enum AuthorizationErrorCode {
 
   /// The authorization attempt failed for an unknown reason.
   unknown,
+
+  /// The credential export request failed.
+  credentialExport,
+
+  /// The credential import request failed.
+  credentialImport,
+
+  /// This error should only be returned when specifying @c excludedCredentials on a public key credential registration request.
```

Comment:
What does `@c` mean here?

Reply:
Hmm, weird, this is also similar in the official docs, but I wonder why this case is treated separately from the others?

![CleanShot 2025-04-05 at 14 47 24@2x](https://github.com/user-attachments/assets/5aeb4f9c-4741-414a-bda7-aa27f4ecea7a)